src/effects/effectmanifest.h

```diff
@@ -76,6 +77,14 @@ class EffectManifest final {
         return m_description;
     }
 
+    const bool& isVisible() const {
+        return m_isVisible;
+    }
+
+    void setVisibility(const bool value) {
+        m_isVisible = value;
+    }
```

Comment:
Nothing else in the effect manifest is changed during the runtime of Mixxx. I think it would be better for EffectsManager or EffectsBackend to hold the information of which effects are blacklist ed.

Reply:
I think be better to store which effects are hidden in EffectsManager than in EffectsBackend.